poloTrader.py

```diff
@@ -105,6 +125,8 @@ def compute_portfolio_stats(normed_vals, rfr = 0, sf = 252):
 	return cr, adr, sddr, sr
 	
 def get_max_close(normed_vals):
+
+	logging.debug('Entered get_max_close(%s, %s, %s)', str(normed_vals))
```

Comment:
I'm getting a crash here saying "TypeError: not enough arguments for format string"